### PR TITLE
Adjust the file list for amqp.

### PR DESCRIPTION
**Pull request by anthonyvercolano**
No longer used the internal.h file

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anthonyvercolano__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (toercola). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: c/iothub_client/build/iothub_amqp_transport_filelist.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 ..\inc\iothubtransportamqp.h
 ..\src\iothubtransportamqp.c
-..\inc\iothub_client_amqp_internal.h
```
Comment (anthonyvercolano) [inline c/iothub_client/build/iothub_amqp_transport_filelist.txt]: But I like this header file.  I created it!

Comment (zolvarga) [inline c/iothub_client/build/iothub_amqp_transport_filelist.txt]: Tony, it won't work!

File: c/iothub_client/build/iothub_amqp_transport_filelist.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 ..\inc\iothubtransportamqp.h
 ..\src\iothubtransportamqp.c
```
Comment (tameraw) [inline c/iothub_client/build/iothub_amqp_transport_filelist.txt]: what about this one?

**Pull request closed without merging**